### DISCUSSION TITLE
移除集群 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1061] [Announcements]
# 移除集群 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/cluster/remove/

**Comment by liyuerich** [comment]
移除  还是  移出？

**Comment by windsonsea** [comment]
应是 移除，已提交前端 UI PR。

merge 后重新截图。
@Michelle951

**Comment by windsonsea** [comment]
UI 的问题，应该是 `移除`
